lib/components/Widgets/WidgetContainer/index.stories.tsx

```diff
@@ -77,3 +77,29 @@ export const MultipleContent: Story = {
     ],
   },
 }
+
+export const MultipleContentWihNulls: Story = {
```

Comment:
This should probably be just a spec, I don't think it should be part of the documentation, right? It's extremely easy, check out any `.spec.ts` examples.

Reply:
done